### DISCUSSION TITLE
scraping data from facebook page

### DISCUSSION THREAD

**Discussion by kegould** [kevinzg/facebook-scraper#326] [Q&A] [answered]
Hi, was wondering if there's a way to extract the number of likes/followers a page on facebook has (i.e. not just the posts or not data from a profile.) Thanks!

**Comment by neon-ninja** [comment, marked as answer]
Sure - this commit (https://github.com/kevinzg/facebook-scraper/commit/adfd6da732077730d4935e22854d17aa9f2f667b) adds a new function called get_page_info. Usage:

```python
from pprint import pprint
from facebook_scraper import *
pprint(get_page_info("Nintendo"))
```

output

```python
{'dateCreated': '2011-06-01T15:23:50-0700',
 'followers': 5469624,
 'identifier': 119240841493711,
 'image': 'https://scontent.fhlz2-1.fna.fbcdn.net/v/t1.18169-1/13138816_1025939437490509_1827627750936743506_n.png?_nc_cat=1&ccb=1-3&_nc_sid=8d0d6e&_nc_ohc=xam_jLPoWYgAX8lPUCE&_nc_ht=scontent.fhlz2-1.fna&oh=8134938ede2a41a4250ac81e95cfc235&oe=60E61E52',
 'likes': 5410941,
 'name': 'Nintendo',
 'sameAs': 'http://www.nintendo.com',
 'type': 'Organization',
 'url': 'https://www.facebook.com/Nintendo/'}
```

**Reply by brandon-scott-pritchard** [reply]
I've been trying to use this function, but it only ever returns likes.

**Reply by neon-ninja** [reply]
What pages are you having this problem with?

**Comment by brandon-scott-pritchard** [comment]
businesses = ['bpattorney',
                'UtahDivorceAttorneyDavidPedrazas',
                'Bobby-Dale-Barina-Attorney-at-Law-271503059621431',
                'FamilyLawRights',
                'diyfamilylaw',
                'salcidolaw',
                'GilstrapLawoffice',
                'OneilWysocki',
                'LatinaLawyer',
                'putmanlaw',
                'JerseyCityDivorceLawyer',
                'leannetownsendlife',
                'caflawgroup']

I pulled each of these page names from the URL to be safe. Some return just likes; some return nothing at all.

**Reply by neon-ninja** [reply]
Try pass cookies as per the readme
